Kconfig

```diff
@@ -956,6 +956,10 @@ menu "LVGL configuration"
 		config LV_USE_FONT_PLACEHOLDER
 			bool "Enable drawing placeholders when glyph dsc is not found"
 			default y
+
+		config LV_TEST_FONT_PATH
```

Comment:
Since this configuration is only used by the benchmark, please move this configuration below the `LV_USE_DEMO_BENCHMARK` configuration, rename it to `LV_DEMO_BENCHMARK_FREETYPE_FONT_PATH` and add the dependency `LV_USE_FREETYPE`.